libraries/src/HTML/Helpers/Grid.php

```diff
@@ -44,7 +44,7 @@ abstract class Grid
     public static function sort($title, $order, $direction = 'asc', $selected = '', $task = null, $newDirection = 'asc', $tip = '', $form = null)
     {
         HTMLHelper::_('behavior.core');
-        HTMLHelper::_('bootstrap.popover', '.hasPopover', ['trigger' => 'hover focus']);
+        HTMLHelper::_('bootstrap.tooltip', '.hasTooltip', ['trigger' => 'hover focus']);
```

Comment:
```suggestion
        HTMLHelper::_('bootstrap.tooltip', '.hasTooltip');
```
These are default triggers so not necessary to specify them.